plotter/Plotter.cpp

```diff
@@ -167,15 +167,15 @@ void Plotter::Plot()
 {
     bool config = counter == 0;
 
-    Serial.print( "{\"" + TIME_KEY + "\":" ); Serial.print( millis() );
+    Serial.print( "{\""); Serial.print(TIME_KEY); Serial.print("\":" ); Serial.print( millis() );
```

Comment:
@Chritzel - for example this line would be:
```
Serial.print( "{\"" ); Serial.print( TIME_KEY ); Serial.print( "\":" ); Serial.print( millis() );
```

Reply:
Got it, added it.